### PR TITLE
Adding a Conclusion episode

### PR DESCRIPTION
**Pull request by ledm**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [ ] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [ ] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes {#40}

### CODE REVIEW

File: _episodes/conclusions.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+exercises: 0
+questions:
+- "What do I do now?"
+- "Where can I get help?"
+- "What if I find a bug?"
+- "Where can I find more information about ESMValtool?"
+- "How can I cite ESMValtool?"
+
+objectives:
+- "breathe - you're finished now!"
```
Comment (hb326) [inline _episodes/conclusions.md:13]: :smile:

File: _episodes/conclusions.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+### Where can I get more information on ESMValTool?
+
+Read the docs page: https://esmvaltool.readthedocs.io/
+
+Technical description paper: https://doi.org/10.5194/gmd-13-1179-2020 
+
+Source code (ESMValTool): https://github.com/ESMValGroup/ESMValTool
+
+Source code (ESMValCore ): https://github.com/ESMValGroup/ESMValCore 
+
```
Comment (hb326) [inline _episodes/conclusions.md:44]: Do we want to add the website here as well? 
https://www.esmvaltool.org/

File: _episodes/conclusions.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+### Where can I get more help?
+
+There are lots of resources available for helping you use ESMValTool. 
+
+If you get stuck, a great starting point is to create a new issue. [Link to ep 10]
+
```
Comment (hb326) [inline _episodes/conclusions.md:51]: I think we might have to provide some more text than only the link to ep 10. After all, we say that there are lots of resources available. 
Can we add here that many problems are discussed in current and old issues on Github? And that people should check these out?
Maybe we could add the link to where people can sign up for the ESMValTool info email (I think something like that exists, right?)
Maybe add the link to a FAQ page?
I can't remember if we decided to provide a support email address somewhere?

File: _episodes/conclusions.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+
+### What if I find a bug? 
+
+If you find a bug, please report it back to the ESMValTool team. This will help us fix it so that you can continue working, but also it means that ESMValTool will be more stable for everyone else as well. 
+
+Please create a new issue using this page: https://github.com/ESMValGroup/ESMValTool/issues
```
Comment (hb326) [inline _episodes/conclusions.md:58]: Maybe rephrase to "To report a bug, please create..." that it is clear that for bug-reporting you have to create an issue.

File: _episodes/conclusions.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+In your issue, please describe the problem as clearly and as completely as possible. 
+
+
+### How do I cite ESMValTool?
+
+Please use the following reference:
+
+Righi, M., Andela, B., Eyring, V., Lauer, A., Predoi, V., Schlund, M., Vegas-Regidor, J., Bock, L., Brötz, B., de Mora, L., Diblen, F., Dreyer, L., Drost, N., Earnshaw, P., Hassler, B., Koldunov, N., Little, B., Loosveldt Tomas, S., and Zimmermann, K.: Earth System Model Evaluation Tool (ESMValTool) v2.0 – technical overview, Geosci. Model Dev., 13, 1179–1199, https://doi.org/10.5194/gmd-13-1179-2020, 2020.
+
```
Comment (hb326) [inline _episodes/conclusions.md:68]: Do we want to add the other three ESMValTool v2.0 papers here as well? I am actually not sure how the ESMValTool v2.0 is cited correctly, if Mattia's paper is enough, or if all four papers need to be cited. I will check with Axel.

File: _episodes/conclusions.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+questions:
+- "What do I do now?"
+- "Where can I get help?"
+- "What if I find a bug?"
+- "Where can I find more information about ESMValtool?"
+- "How can I cite ESMValtool?"
+
+objectives:
+- "breathe - you're finished now!"
+- "Congratulations & Thanks!”
```
Comment (LisaBock) [inline _episodes/conclusions.md:14]: Last character is wring formatted...

File: _episodes/conclusions.md (hunk 5 above)
Comment (ledm) [reply _episodes/conclusions.md:68]: I don't think that we do. If people directly use those papers, then yes. However, if you just want to cite ESMValTool, best to stick with the technical description. 

I've added the other papers to the additional information section.

File: _episodes/conclusions.md (hunk 3 above)
Comment (hb326) [reply _episodes/conclusions.md:51]: Ha! We DO have an email address (a listserve address): esmvaltool@listserv.dfn.de

File: _episodes/conclusions.md (hunk 6 above)
Comment (ledm) [reply _episodes/conclusions.md:14]: Fixed

File: _episodes/conclusions.md (hunk 4 above)
Comment (ledm) [reply _episodes/conclusions.md:58]: Resolved.

File: _episodes/conclusions.md (hunk 2 above)
Comment (ledm) [reply _episodes/conclusions.md:44]: Resolved.

File: _episodes/conclusions.md (hunk 3 above)
Comment (ledm) [reply _episodes/conclusions.md:51]: I'm not sure about this. It's always been an open discussion. @valeriupredoi @bouweandela, any thoughts on what we should put in the " Where can I get more help?" section of the conclusion page.

File: _episodes/conclusions.md (hunk 5 above)
Comment (hb326) [reply _episodes/conclusions.md:81]: Ok. Fine with me.

File: _episodes/conclusions.md (hunk 3 above)
Comment (ledm) [reply _episodes/conclusions.md:60]: We should add the list here.

File: _episodes/conclusions.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+> - [Read the docs page](https://esmvaltool.readthedocs.io/)
+> - [ESMValTool home page](https://www.esmvaltool.org/)
+> - [Technical description paper](https://doi.org/10.5194/gmd-13-1179-2020)
+> - [Source code (ESMValTool)](https://github.com/ESMValGroup/ESMValTool)
+> - [Source code (ESMValCore )](https://github.com/ESMValGroup/ESMValCore)
+{: .callout}
+
+
+Additional publications:
+- Lauer, A., Eyring, V., Bellprat, O., Bock, L., Gier, B. K., Hunter, A., Lorenz, R., Pérez-Zanón, N., Righi, M., Schlund, M., Senftleben, D., Weigel, K., and Zechlau, S.: Earth System Model Evaluation Tool (ESMValTool) v2.0 – diagnostics for emergent constraints and future projections from Earth system models in CMIP, Geosci. Model Dev. Discuss., https://doi.org/10.5194/gmd-2020-60, in review, 2020.
```
Comment (ledm) [inline _episodes/conclusions.md:56]:
```suggestion
- Lauer, A., Eyring, V., Bellprat, O., Bock, L., Gier, B. K., Hunter, A., Lorenz, R., 
Pérez-Zanón, N., Righi, M., Schlund, M., Senftleben, D., Weigel, K., and Zechlau, S.:
Earth System Model Evaluation Tool (ESMValTool) v2.0 – diagnostics
for emergent constraints and future projections from Earth system models in CMIP,
Geosci. Model Dev. Discuss.,
[https://doi.org/10.5194/gmd-2020-60](https://doi.org/10.5194/gmd-2020-60), in review, 2020.
```

File: _episodes/conclusions.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+> - [ESMValTool home page](https://www.esmvaltool.org/)
+> - [Technical description paper](https://doi.org/10.5194/gmd-13-1179-2020)
+> - [Source code (ESMValTool)](https://github.com/ESMValGroup/ESMValTool)
+> - [Source code (ESMValCore )](https://github.com/ESMValGroup/ESMValCore)
+{: .callout}
+
+
+Additional publications:
+- Lauer, A., Eyring, V., Bellprat, O., Bock, L., Gier, B. K., Hunter, A., Lorenz, R., 
+Pérez-Zanón, N., Righi, M., Schlund, M., Senftleben, D., Weigel, K., and Zechlau, S.:
```
Comment (ledm) [inline _episodes/conclusions.md:57]:
```suggestion
  Pérez-Zanón, N., Righi, M., Schlund, M., Senftleben, D., Weigel, K., and Zechlau, S.:
```

File: _episodes/conclusions.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+for emergent constraints and future projections from Earth system models in CMIP,
+Geosci. Model Dev. Discuss.,
+[https://doi.org/10.5194/gmd-2020-60](https://doi.org/10.5194/gmd-2020-60), in review, 2020.
+
+- Eyring, V., Bock, L., Lauer, A., Righi, M., Schlund, M., Andela, B., Arnone, E., Bellprat, O., Brötz, B., Caron, L.-P., Carvalhais, N., Cionni, I., Cortesi, N., Crezee, B., Davin, E., Davini, P., Debeire, K., de Mora, L., Deser, C., Docquier, D., Earnshaw, P., Ehbrecht, C., Gier, B. K., Gonzalez-Reviriego, N., Goodman, P., Hagemann, S., Hardiman, S., Hassler, B., Hunter, A., Kadow, C., Kindermann, S., Koirala, S., Koldunov, N. V., Lejeune, Q., Lembo, V., Lovato, T., Lucarini, V., Massonnet, F., Müller, B., Pandde, A., Pérez-Zanón, N., Phillips, A., Predoi, V., Russell, J., Sellar, A., Serva, F., Stacke, T., Swaminathan, R., Torralba, V., Vegas-Regidor, J., von Hardenberg, J., Weigel, K., and Zimmermann, K.: ESMValTool v2.0 – Extended set of large-scale diagnostics for quasi-operational and comprehensive evaluation of Earth system models in CMIP, Geosci. Model Dev. Discuss., https://doi.org/10.5194/gmd-2019-291, in review, 2019.
```
Comment (ledm) [inline _episodes/conclusions.md:63]:
```suggestion
- Eyring, V., Bock, L., Lauer, A., Righi, M., Schlund, M., Andela, B., Arnone, E., 
  Bellprat, O., Brötz, B., Caron, L.-P., Carvalhais, N., Cionni, I., Cortesi, N., Crezee, B., 
  Davin, E., Davini, P., Debeire, K., de Mora, L., Deser, C., Docquier, D., Earnshaw, P., 
  Ehbrecht, C., Gier, B. K., Gonzalez-Reviriego, N., Goodman, P., Hagemann, S., 
  Hardiman, S., Hassler, B., Hunter, A., Kadow, C., Kindermann, S., Koirala, S., 
  Koldunov, N. V., Lejeune, Q., Lembo, V., Lovato, T., Lucarini, V., Massonnet, F., 
  Müller, B., Pandde, A., Pérez-Zanón, N., Phillips, A., Predoi, V., Russell, J., Sellar, A., 
  Serva, F., Stacke, T., Swaminathan, R., Torralba, V., Vegas-Regidor, J., von Hardenberg, J., 
  Weigel, K., and Zimmermann, K.: 
  ESMValTool v2.0 – Extended set of large-scale diagnostics for quasi-operational 
  and comprehensive evaluation of Earth system models in CMIP, 
  Geosci. Model Dev. Discuss., 
  [https://doi.org/10.5194/gmd-2019-291](https://doi.org/10.5194/gmd-2019-291),
   in review, 2019.
```

File: _episodes/conclusions.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+  Müller, B., Pandde, A., Pérez-Zanón, N., Phillips, A., Predoi, V., Russell, J., Sellar, A., 
+  Serva, F., Stacke, T., Swaminathan, R., Torralba, V., Vegas-Regidor, J., von Hardenberg, J., 
+  Weigel, K., and Zimmermann, K.: 
+  ESMValTool v2.0 – Extended set of large-scale diagnostics for quasi-operational 
+  and comprehensive evaluation of Earth system models in CMIP, 
+  Geosci. Model Dev. Discuss., 
+  [https://doi.org/10.5194/gmd-2019-291](https://doi.org/10.5194/gmd-2019-291),
+   in review, 2019.
+
+- Eyring, V., Righi, M., Lauer, A., Evaldsson, M., Wenzel, S., Jones, C., Anav, A., Andrews, O., Cionni, I., Davin, E. L., Deser, C., Ehbrecht, C., Friedlingstein, P., Gleckler, P., Gottschaldt, K.-D., Hagemann, S., Juckes, M., Kindermann, S., Krasting, J., Kunert, D., Levine, R., Loew, A., Mäkelä, J., Martin, G., Mason, E., Phillips, A. S., Read, S., Rio, C., Roehrig, R., Senftleben, D., Sterl, A., van Ulft, L. H., Walton, J., Wang, S., and Williams, K. D.: ESMValTool (v1.0) – a community diagnostic and performance metrics tool for routine evaluation of Earth system models in CMIP, Geosci. Model Dev., 9, 1747–1802, https://doi.org/10.5194/gmd-9-1747-2016, 2016.
```
Comment (ledm) [inline _episodes/conclusions.md:78]:
```suggestion
- Eyring, V., Righi, M., Lauer, A., Evaldsson, M., Wenzel, S., Jones, C., Anav, A.,
  Andrews, O., Cionni, I., Davin, E. L., Deser, C., Ehbrecht, C., Friedlingstein, P., 
  Gleckler, P., Gottschaldt, K.-D., Hagemann, S., Juckes, M., Kindermann, S., 
  Krasting, J., Kunert, D., Levine, R., Loew, A., Mäkelä, J., Martin, G., Mason, E., 
  Phillips, A. S., Read, S., Rio, C., Roehrig, R., Senftleben, D., Sterl, A., 
  van Ulft, L. H., Walton, J., Wang, S., and Williams, K. D.: 
  ESMValTool (v1.0) – a community diagnostic and performance metrics tool for 
  routine evaluation of Earth system models in CMIP, 
  Geosci. Model Dev., 9, 1747–1802, 
  [https://doi.org/10.5194/gmd-9-1747-2016](https://doi.org/10.5194/gmd-9-1747-2016),
  2016.
```

**Comment by Peter9192** [comment]
Thanks guys!
